dashboard/package.json

```diff
@@ -37,6 +19,7 @@
     "google-protobuf": "^3.17.3",
     "history": "^4.10.1",
     "immutable": "^4.0.0-rc.14",
+    "isomorphic-form-data": "^2.0.0",
```

Comment:
So it sounds from @dlaloue-vmware 's comments that this dependency is required by swagger-ui-react but wasn't declared as a dependency by them? Or something similar: https://github.com/swagger-api/swagger-ui/issues/7436

Hah, although they mention there that: "On Thursday 29.7.2021 there will a new release of swagger-ui-react that will fix this issue automatically by updating to it.". Not sure if we wait, but if not, let's at least include a #TODO to remove it refering to the swagger-ui issue ?

Reply:
While writing the message, they have just bumped up the version: https://github.com/swagger-api/swagger-ui/releases/tag/v3.51.2